3D-microservices-observability/node-js-app/beachshirt/controllers/stylingController.js

```diff
@@ -0,0 +1,47 @@
+/**
+* Driver for styling service which manages different styles of shirts and takes orders for a shirts
+* of a given style.
+* @author Yogesh Prasad Kurmi (ykurmi@vmware.com)
+*/
+const request = require('request');
+const uuidv1 = require('uuid/v1');
+
+module.exports = function(app, config, log){
+    app.get('/style', function(req, res){
+        res.json(
+            [
+                {
+                    "name": "Wavefront",
+                    "url": "WavefrontURL"
```

Comment:
is this supposed to be hard-coded?

Reply:
As of now, I have hardcoded it. In the future, if there is any plan for getting it from the database I will change it.